### DISCUSSION TITLE
Self Host Auth Panel & Documentation

### DISCUSSION THREAD

**Discussion by orion-supernova** [supabase/supabase#29001] [Feature Requests]
There is no documentation for OAuth with self hosted supabase. Im trying to implement Sign in With Apple with my self hosted supabase and I couldn't find any docs.

Besides that, It would be great to configure them inside Auth screen inside Supabase. The config page is not visible directly. I use them with going to url:
http://localhost:8000/project/default/auth/providers

**Comment by Oliver-Turp** [comment]
Did you get this working? I'm also struggling with this
